aten/src/ATen/cuda/cub.cuh

```diff
@@ -65,35 +65,30 @@ static inline void sort_keys(
   if (descending) {
     CUB_WRAPPER(NO_ROCM(detail)::cub::DeviceRadixSort::SortKeysDescending,
       keys_in_, keys_out_, n,
-      start_bit, end_bit, c10::cuda::getCurrentCUDAStream());
+      begin_bit, end_bit, c10::cuda::getCurrentCUDAStream());
   } else {
     CUB_WRAPPER(NO_ROCM(detail)::cub::DeviceRadixSort::SortKeys,
       keys_in_, keys_out_, n,
-      start_bit, end_bit, c10::cuda::getCurrentCUDAStream());
+      begin_bit, end_bit, c10::cuda::getCurrentCUDAStream());
   }
 }
 
 template<typename key_t, typename value_t>
 static inline void sort_pairs(
     const key_t *keys_in, key_t *keys_out,
     const value_t *values_in, value_t *values_out,
-    int64_t n, bool descending=false, int64_t start_bit=0, int64_t end_bit=sizeof(key_t)*8
+    int64_t n, bool descending=false, int64_t begin_bit=0, int64_t end_bit=sizeof(key_t)*8
 ) {
   using key_t_ = typename cuda_type<key_t>::type;
   using value_t_ = typename cuda_type<value_t>::type;
 
   auto allocator = c10::cuda::CUDACachingAllocator::get();
   c10::DataPtr keys_out_owner;
-  c10::DataPtr values_out_owner;
 
   if (keys_out == nullptr) {
     keys_out_owner = allocator->allocate(n * sizeof(key_t));
     keys_out = reinterpret_cast<key_t *>(keys_out_owner.get());
   }
-  if (values_out == nullptr) {
```

Comment:
`values_out` will never be nullptr. If it is, then people should use `sort_keys` instead of `sort_pairs`